actors/evm/src/interpreter/precompiles/mod.rs

```diff
@@ -61,8 +61,8 @@ const fn gen_native_precompiles<RT: Runtime>() -> [PrecompileFn<RT>; 4] {
     }
 }
 
-pub fn is_reserved_precompile_address(addr: [u8; 20]) -> bool {
-    let [prefix, middle @ .., index] = addr;
+pub fn is_reserved_precompile_address(addr: &U256) -> bool {
```

Comment:
Rather than change this to a U256, I'd change it to an `EthAddress`. It'll be faster and will simplify a lot of code.

Reply:
yep.